.coveragerc

```diff
@@ -1,7 +1,7 @@
 [run]
-source = energyplus_rmd
+source = energyplus_rpd
```

Comment:
I hope you don't imagine this changing often.  But while we are changing it...I wonder about energyplus_rulesets rather than an obscure acronym?

Reply:
"energyplus_rulesets" is probably not quite enough information and might be confusing for something that implemented a ruleset. I will think about this in the future.